app/scripts/modules/core/src/healthCounts/healthCounts.less

```diff
@@ -22,11 +27,6 @@ health-counts {
     float: right;
     padding-left: 5px;
   }
-  &.no-float {
```

Comment:
Not sure - it's only used right now in the Angular world - should I just duplicate it in both places?